packages/sn-filter-pane/src/qae/object-definition.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * @namespace properties
+ * @entry
+ */
+const objectDefinition = () => {
+  /**
+   * @lends properties
+   */
+  const definition = {
+    /**
+     * Current version of this generic object definition
+     * @type {string}
+     * @default
+     */
+    version: process.env.PACKAGE_VERSION,
+    /**
+     * Show visualization details toggle
+     * @type {boolean=}
+     * @default
+     */
+    showDetails: true,
+    /**
+     * Show title for the visualization.
+     * @type {boolean=}
+     * @default
+     */
+    showTitles: false,
+    /**
+     * Visualization title.
+     * @type {(string|StringExpression)=}
+     * @default
+     */
+    title: '',
+    /**
+     * Visualization subtitle.
+     * @type {(string|StringExpression)=}
+     * @default
+     */
+    subtitle: '',
+    /**
+     * Visualization footnote.
+     * @type {(string|StringExpression)=}
+     * @default
+     */
+    footnote: '',
+    /**
+     * Child object definition
+     * @type {ChildListDef}
+     * @default {qDef:{qListObjectDef:{}}}
+     */
+    qChildListDef: {
+      qDef: {
+        qListObjectDef: {},
+      },
+    },
```

Comment:
we should probably clean this up
I think it should be
```suggestion
    /**
     * Child object definition
     * @type {ChildListDef}
     * @default {}
     */
    qChildListDef: {},
```